src/features/callAssignments/apiTypes.ts

```diff
@@ -12,6 +13,7 @@ export interface CallAssignmentStats {
   callBackLater: number;
   calledTooRecently: number;
   done: number;
+  isStale: boolean;
```

Comment:
I think this solution is good (duh!) in theory, but should be executed some other way. We can save this for a refactor later, or do it now. Not sure what is best?

In this code, domain data (like `callBackLater`, `target` and `title`) is mixed with loading/caching support meta-data (`isStale, isLoading`) in the same object. I think it should be separated, so that there's a "container" object (defined elsewhere) with the meta-data, and a data field for the actual data, i.e. something like this:

```ts
interface CachedData<T> {
  isLoading: boolean;
  isStale: boolean;
  data: T;
}
```

Reply:
Yes, I felt this scratch at my brain when we did it, but accepted it as a "for now" solution. I think to separate it out into a different issue, and think about it there, so we can put the basic functionality of this pr in the books, and break out the "bigger question" into a separate pr.